internal/bminventory/inventory.go

```diff
@@ -4438,12 +4439,7 @@ func (b *bareMetalInventory) V2GetPresignedForClusterFiles(ctx context.Context,
 }
 
 func (b *bareMetalInventory) GetPresignedForClusterFiles(ctx context.Context, params installer.GetPresignedForClusterFilesParams) middleware.Responder {
-	presigned, err := b.getPresignedForClusterFiles(ctx, &params.ClusterID, params.FileName, params.AdditionalName, params.HostID, params.LogsType)
-	if err != nil {
-		return err
-	}
-
-	return installer.NewGetPresignedForClusterFilesOK().WithPayload(presigned)
+	return common.GenerateErrorResponder(errors.New(APINotImplemented))
```

Comment:
Isn't that 501? https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/501

Reply:
Yes, Done.